src/Http/Http.Abstractions/src/Metadata/IStatusCodePagesMetadata.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Http.Metadata;
+
+/// <summary>
+/// Defines a contract used to specify metadata for skipping the StatusCodePage
+/// middleware in <see cref="Endpoint.Metadata"/>.
+/// </summary>
+public interface ISkipStatusCodePagesMetadata
+{
+    /// <summary>
+    /// Gets whether or not status code pages are enabled for an endpoint.
+    /// </summary>
+    bool Enabled { get; }
```

Comment:
Is the bool needed? I don't see any usage of it in tests, plus isn't setting it to true equivalent to just not applying the attribute?

Reply:
We talked about this during API review and decided to remove it as well.